sites/all/modules/ar/ar_assessments/js/ar_assessments.js

```diff
@@ -405,7 +405,7 @@ Drupal.behaviors.arAssessmentsAssessments = {
       },
 
       defaultRoute: function (actions) {
-        this.navigate('list/1', {trigger: true});
```

Comment:
@amyvbenson Unless this changed recently, the default route should normally be the list view